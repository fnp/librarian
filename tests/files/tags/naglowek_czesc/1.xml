<autor_utworu>Juliusz Słowacki</autor_utworu>
<nazwa_utworu>Beniowski</nazwa_utworu>

<naglowek_czesc>PIEŚŃ I</naglowek_czesc>

<strofa>
Za panowania króla Stanisława/
<wers_wciety>Mieszkał ubogi szlachcic na Podolu,</wers_wciety>/
Wysoko potem go wyniosła sława;/
<wers_wciety>Szczęścia miał mało w życiu, więcej bolu;</wers_wciety>/
Albowiem była to epoka krwawa,/
<wers_wciety>I kraj był cały na rumaku, w polu;</wers_wciety>/
Łany, ogrody leżały odłogiem,/
Zaraza stała u domu za progiem.
</strofa>
