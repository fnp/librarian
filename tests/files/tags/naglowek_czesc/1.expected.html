<div id="book-text">
<div id="toc">
<h2>Spis treści</h2>
<ol>
<a href="#s1" class="anchor">PIEŚŃ I</a><li>
</ol>
</div>
<h1>
<span class="wl author" id="i1">Juliusz Słowacki</span><span class="wl title" id="i2">Beniowski</span>
</h1>
<h2 class="wl" id="s1">PIEŚŃ I</h2>
<div class="stanza" id="i3">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Za panowania króla Stanisława</div>
<div class="wl verse verse-indent verse-indent-1" id="f2">Mieszkał ubogi szlachcic na Podolu,</div>
<div class="wl verse" id="f3">Wysoko potem go wyniosła sława;</div>
<div class="wl verse verse-indent verse-indent-1" id="f4">Szczęścia miał mało w życiu, więcej bolu;</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">Albowiem była to epoka krwawa,</div>
<div class="wl verse verse-indent verse-indent-1" id="f6">I kraj był cały na rumaku, w polu;</div>
<div class="wl verse" id="f7">Łany, ogrody leżały odłogiem,</div>
<div class="wl verse" id="f8">Zaraza stała u domu za progiem.</div>
</div>
</div>
